src/ngMock/angular-mocks.js

```diff
@@ -1884,7 +1885,12 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
   $httpBackend.verifyNoOutstandingRequest = function(digest) {
     if (digest !== false) $rootScope.$digest();
     if (responses.length) {
-      throw new Error('Unflushed requests: ' + responses.length);
+      var unflushedDescriptions = [];
+      for (var i = 0, len = responses.length; i < len; ++i) {
+        unflushedDescriptions.push(responses[i].description);
+      }
```

Comment:
Nit: `var unflushedDescriptions = responses.map(function(res) { return res.description; });` is shorter.